docs/sections/user_guide/uw_yaml/forecast_yaml.rst

```diff
@@ -0,0 +1,232 @@
+
+The configuration files required by the ufs-weather-model are documented :weather-model-io:`here<model-configuration-files>`.
+
+The ``forecast:`` section
+-------------------------
+
+The ``forecast:`` section describes the specifics of the FV3 atmosphere forecast component.
+
+.. code-block:: yaml
+
+  forecast:
+    cycle-dependent:
+      INPUT/gfs_data.nc: /path/to/gfs_data.nc
+      INPUT/sfc_data.nc: /path/to/sfc_data.nc
+      INPUT/gfs_ctrl.nc: /path/to/gfs_ctrl.nc
+    diag_table:
+      template_file: /path/to/diag_table/template/file
+    domain: regional
+    executable: fv3.exe
+    fd_ufs:
+      base_file: /path/to/base/fd_ufs.yaml
+    field_table:
+      base_file: /path/to/field_table.yaml
+      update_values:
+        liq_wat:
+          longname: cloud water mixing ratio
+          units: kg/kg
+          profile_type:
+            name: fixed
+            surface_value: 2.0
+    length: 12
+    model_configure:
+      base_file: /path/to/base/model_configure
+      update_values:
+        write_dopost: .false.
+    namelist:
+      base_file: /path/to/base/input.nml
+      update_values:
+        fv_core_nml:
+          k_split: 2
+          n_split: 6
+    nems.configure:
+      template_file: /path/to/template/nems.configure
+    run_dir: /path/to/forecast/run/directory/{yyyymmddhh}
+    static:
+      fv3.exe: /path/to/executable/ufs_model
+      INPUT/grid_spec.nc: /path/to/grid_spec.nc
+      ...
+      data_table: /path/to/data_table
+      eta_micro_lookup.data: /path/to/noahmptable.dat
+      noahmptable.tbl: /path/to/noahmptable.tbl
+
+.. _updating_values:
+
+Updating Values
+^^^^^^^^^^^^^^^
+
+Many of the sections describe the configuration files needed by the FV3 Forecast model, i.e. namelist, fd_ufs, model_configure. The ``base_file:`` entry is required so that the file will be staged. Modifications to the base file can be made at run time by providing an ``update_values:`` block.
+
+To ensure the correct values are updated, all keys that act as section headers above the entry to be updated need to be provided in the order in which they appear in the base file. Multiple entries within a block may be updated and they need not follow the same order as those in the base file. For example, the base file named ``people.yaml`` may contain:
+
+.. code-block:: yaml
+
+   person:
+     age: 19
+     address:
+       city: Boston
+       number: 12
+       state: MA
+       street: Acorn St
+     name: Jane
+
+Then the entries under the YAML section would edit this base file with the entries:
+
+.. code-block:: yaml
+
+   base_file: people.yaml
+   update_values:
+     person:
+       address:
+         street: Main St
+         number: 99
+
+Rendering Template Files
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+Requires a path to a template file in the ``template_file:`` entry. There is no option to add values in the YAML. Instead, the driver is programmed to enter necessary values for the template.
```

Comment:
Here, I was looking for a way to say that the driver owns the values needed for the templates. The user doesn't provide them directly as they might in the `update_values:` entries for a YAML or namelist file.

Reply:
That makes me wonder: If there's no extra information to be provided, could we just have

```
diag_table: /path/to/diag_table/template/file
```

instead of

```
diag_table:
  template_file: /path/to/diag_table/template/file
```

? Is the extra level providing anything except potential confusion such that we have to document for the user (I'm paraphrasing what I'm now guessing is the purpose of this bit of doc), "Don't worry that you're not seeing extra keys and values here!"

Reply:
It was originally meant to provide a visual cue that it was treated as a template. And it might be helpful to one day build in an option to override values provided to the template. (I'm screaming YAGNI at myself right now).

Reply:
I'm glad I don't have to :)